.circleci/config.yml

```diff
@@ -54,6 +54,12 @@ jobs:
             python --version
             coverage run -m unittest discover
 
+      - run: &run-cpp-tests
```

Comment:
I think this should probably be a separate job, rather than added to the existing python jobs.

Reply:
I've added one linux job. Will switch to CMake and add additional jobs (for Windows, OSX, and additional compilers) in a subsequent PR.